cisagov/getgov
# Issue #2595: update domain request view only pages - [ZA]

## Ticket

Resolves #2595


## Changes


- Changes which date(s) display on the viewonly pages for which state. For instance, withdrawn now has a `Withdrawn on:` date.
- Fixes a bug with cisa representative wherein it would always show `no` even if one did in fact exist.
- Fixes a bug where the domain requests header in portfolio mode would show both `domains` and `domain requests` as active
- Adds a breadcrumb instead of a back button for these pages



## Context for reviewers
**Important: ** When viewing these changes in portfolio view, make sure that the domain requests that you add a portfolio have yourself set as the creator. We have not yet implemented a viewonly view, as that is a follow-on ticket. This means that you will only see these dates on the "manage" page for now.

To see your portfolio, you will need the following:
1. The waffle flags `organization_feature` and `organization_requests` enabled ("everyone" set to "yes)
2. A portfolio added on multiple domain requests
3. A new `User portfolio permission` record with yourself added as an admin. This can be found on that table.
4. A `started` domain request that you yourself created (aka using the start a new domain request button)

Note: You can technically view our viewonly mode for started and action needed domains, but its not yet supported. To do so, you need to copy the domain request id in the url, and navigate to `http://getgov-za.app.cloud.gov/domain-request/{ID}`.

## Setup
You will need to test the following cases:
- The view-only page on a domain request in every status (minus approved).  Ensure that the dates here match the ones on the figma stylistically. Do note that in_review and ineligible will just show `Last updated on` as no update was specified.
- Add a cisa representative to a domain request in view-only mode. Ensure that the cisa rep section doesn't display `no`, but instead displays their name
- Ensure that clicking the view mode of a domain request doesn't cause a header bug where it looks like you are on the domains page, and the domains request page
- Test that the breadcrumb works correctly
- Test that all of the above still apply when in portfolio view


## Code Review Verification Steps

### As the original developer, I have

#### Satisfied acceptance criteria and met development standards


#### Ensured code standards are met (Original Developer)


#### Validated user-facing changes (if applicable)


### As a code reviewer, I have

#### Reviewed, tested, and left feedback about the changes


#### Ensured code standards are met (Code reviewer)


#### Validated user-facing changes as a developer




**Note:** Multiple code reviewers can share the checklists above, a second reviewers should not make a duplicate checklist

### As a designer reviewer, I have

#### Verified that the changes match the design intention


#### Validated user-facing changes as a designer




## Screenshots
